specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json

```diff
@@ -7841,6 +7841,10 @@
         "endCharIndex": {
           "description": "The index within the utterance where the extracted entity ends.",
           "type": "integer"
+        },
+        "role": {
```

Comment:
Please provide evidence of sign-off from Azure API review board.
Please add samples to cover the role.

Reply:
We are currently working with cognitive services on a unified process for APIs review and versioning as currently we don't have any unified pipeline.
Why is the Azure API review mandatory given that we are still working on the process? Also, what kind of evidence is required? can you elaborate?

Reply:
Mentioning @yangyuan

Reply:
Every API design change (including adding new operations/parameters or changing of existing operations/parameters) requires review from Azure API Review Board.

Azure API Review Board can do online review (Teams meeting) or offline review (PR review). For the online review you can forward me the review email thread for the offline review they can just sign-off this PR.

LUIS team has done many reviews with Azure API review board, Carol should familiar with the process.